waspc/examples/todo-typescript/.wasp/out/sdk/wasp/server/actions/index.ts

```diff
@@ -1,36 +1,35 @@
-import prisma from 'wasp/server/dbClient.js'
-import {
-  updateTask as updateTaskUser,
-  createTask as createTaskUser,
-  deleteTasks as deleteTasksUser,
-} from 'wasp/ext-src/actions.js'
+import prisma from 'wasp/server/dbClient'
 
-export type UpdateTask = typeof updateTask
+import { createTask as createTask_ext } from 'wasp/ext-src/task/actions'
+import { updateTask as updateTask_ext } from 'wasp/ext-src/task/actions'
+import { deleteTasks as deleteTasks_ext } from 'wasp/ext-src/task/actions'
```

Comment:
Our `JsImport` machinery doesn't support importing multiple symbols from the same file, so this will have to do for now.